docs/kafka/getting-started-kafka/README.adoc

```diff
@@ -121,11 +121,15 @@ ifndef::qs[]
 endif::[]
 
 .Procedure
-. In the {product-kafka} web console, go to *Streams for Apache Kafka* > *Kafka Instances* and click *Create Kafka instance*.
-. Enter a unique name for the Kafka instance, such as `my-first-kafka-instance`.
-. From the *Cloud region* drop-down menu, select the relevant option. The selection for availability zones is preconfigured as *Single* for a trial instance and *Multi* for a production instance.
+. On the *Kafka Instances* page of the {product-kafka} web console, click *Create Kafka instance*.
+. Complete the *Create a Kafka instance* form to define the following instance details. Some values currently have only one option.
```

Comment:
I'm not sold on the phrase "Some values currently have only one option".
-  Using _currently_ implies to users that this will change in the future. While that might be true, we don't want to make those kinds of promises.
- I think I'd avoid using the term _value_ to represent an instance detail. To me, a _value_ is the number you type in a field or choose from a list. Maybe "Some instance details have only one available option"?

Reply:
Hey, @AedinC. Can you consider applying the comments I previously made here?

Reply:
Yes, I seem to have rewritten over the second commit that incorporated your feedback! Will make those changes again.